content/en/docs/concepts/storage/volumes.md

```diff
@@ -295,13 +295,17 @@ Note that this path is derived from the volume's `mountPath` and the `path`
 keyed with `log_level`.
 
 {{< note >}}
+
 * You must create a [ConfigMap](/docs/tasks/configure-pod-container/configure-pod-configmap/)
   before you can use it.
 
+* By default, a ConfigMap will be mounted as a `readOnly` volume.
```

Comment:
```suggestion
* A ConfigMap is always mounted as `readOnly`.
```

Reply:
You are absolutely correct - https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/configmap/configmap.go#L166

Committing these changes